bigbluebutton-html5/imports/api/2.0/cursor/server/modifiers/updateCursor.js

```diff
@@ -0,0 +1,38 @@
+import Logger from '/imports/startup/server/logger';
+import Cursor from './../../';
+import { check } from 'meteor/check';
+
+export default function updateCursor(meetingId, x = 0, y = 0) {
```

Comment:
A cursor position of (0,0) is a valid position and shouldn't be the default value. A better default is (-1,-1) because that can just be set to "visible=false" or something when rendered.